azurelinuxagent/common/osutil/bigip.py

```diff
@@ -222,7 +222,14 @@ def get_dvd_device(self, dev_dir='/dev'):
             if dvd is not None:
                 return "/dev/{0}".format(dvd.group(0))
         raise OSUtilError("Failed to get dvd device")
-
+    
+    # The linter reports that this function's arguments differ from those
+    # of the function this overrides. This doesn't seem to be a problem, however,
+    # because this function accepts any option that could'be been specified for
+    # the original (and, by forwarding the kwargs to the original, will reject any
+    # option _not_ accepted by the original). Additionally, this method allows us
+    # to keep the defaults for mount_dvd in one place (the original function) instead
+    # of having to duplicate it here as well. 
```

Comment:
But with this signature this function will accept arguments the base function doesn't

Reply:
Sort of; this function basically immediately passes the `kwargs` to its super, which would then error out on invalid input.

I realize that that doesn't completely localize the error to this function, though; I felt like this was better than repeating the defaults here as well, and having to keep both up to date. Maybe this isn't the case, though.

Reply:
ok as is, thanks